components/create_comment/create_comment.jsx

```diff
@@ -240,9 +240,19 @@ class CreateComment extends React.PureComponent {
             createPostErrorId: props.createPostErrorId,
             rootId: props.rootId,
             messageInHistory: props.messageInHistory,
-            draft: state.draft || {...props.draft, caretPosition: props.draft.message.length, uploadsInProgress: []},
         };
 
+        if (state.draft && props.draft && props.draft.message && state.draft.message !== props.draft.message) {
```

Comment:
I wonder about the UX experience here. This opens the door to modifying the input of the user while they are typing (really nice joke plugin).